Docker_Container/README.md

```diff
@@ -0,0 +1,13 @@
+## Just put the whole project inside this folder
+
+1. First install Remote Development from extension
```

Comment:
Add diagrams or be more specific. Show commands to get the job done, line by line. It must be super easy to do.